src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileViewController.swift

```diff
@@ -62,20 +67,145 @@ class AntigenTestProfileViewController: UIViewController, FooterViewHandling, Di
+	private var tableContentObserver: NSKeyValueObservation!
+	private var originalBackgroundImage: UIImage?
+	private var originalShadowImage: UIImage?
 
 	@objc
 	private func backToRootViewController() {
 		didTapDeleteProfile()
 	}
 
+	private func setupNavigationBar() {
+		let logoImage = UIImage(imageLiteralResourceName: "Corona-Warn-App").withRenderingMode(.alwaysTemplate)
+		let logoImageView = UIImageView(image: logoImage)
+		logoImageView.tintColor = .enaColor(for: .textContrast)
+
+		parent?.navigationController?.navigationBar.tintColor = .white
+
+		parent?.navigationItem.titleView = logoImageView
+		parent?.navigationItem.rightBarButtonItem = dismissHandlingCloseBarButton(.contrast)
+
+		// remove previous view controllers from the stack, we will return to the rootViewController by back button here
+		navigationController?.viewControllers = [navigationController?.viewControllers.first, navigationController?.viewControllers.last].compactMap { $0 }
+
+		// keep old images for restoration
+		originalBackgroundImage = navigationController?.navigationBar.backgroundImage(for: .default)
+		originalShadowImage = navigationController?.navigationBar.shadowImage
+
+		// create a transparent navigation bar
+		let emptyImage = UIImage()
+		navigationController?.navigationBar.setBackgroundImage(emptyImage, for: .default)
+		navigationController?.navigationBar.shadowImage = emptyImage
+		navigationController?.navigationBar.isTranslucent = true
+		navigationController?.view.backgroundColor = .clear
+
+		parent?.navigationController?.navigationBar.prefersLargeTitles = false
+		parent?.navigationController?.navigationBar.sizeToFit()
+		parent?.navigationItem.largeTitleDisplayMode = .never
+	}
+
+	private func setupBackground() {
+		backgroundView.gradientHeightConstraint.constant = 300.0
+		backgroundView.translatesAutoresizingMaskIntoConstraints = false
+		view.addSubview(backgroundView)
+
+		tableView.translatesAutoresizingMaskIntoConstraints = false
+		tableView.backgroundColor = .clear
+
+		view.addSubview(tableView)
+
+		NSLayoutConstraint.activate(
+			[
+				backgroundView.topAnchor.constraint(equalTo: view.topAnchor),
+				backgroundView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+				backgroundView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+				backgroundView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+
+				tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+				tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+				tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+				tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+
+			]
+		)
+
+		// observer tableView scrolling to move the background in sync
+		tableContentObserver = tableView.observe(\UITableView.contentOffset, options: .new) { [weak self] _, change in
+			guard let self = self,
+				  let yOffset = change.newValue?.y else {
+				return
```

Comment:
perhaps leave a Log entry here?

Reply:
I added a comment - a log it's helpful here

Reply:
I improved the comment above, a log might not help here